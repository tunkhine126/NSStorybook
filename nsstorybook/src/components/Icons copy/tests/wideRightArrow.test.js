/**
 *
 * Tests for WideRightArrow
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import TestingComponent from 'components/shared/Testing';
import { render } from '@testing-library/react';
// import 'jest-dom/extend-expect'; // add some helpful assertions
import WideRightArrow from '../wideRightArrow';

describe('<WideRightArrow />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(
      <TestingComponent>
        <WideRightArrow childRoute classes={{ arrowSpacing: 'test' }} />
      </TestingComponent>
    );
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <TestingComponent>
        <WideRightArrow />
      </TestingComponent>
    );
    expect(firstChild).toMatchSnapshot();
  });
});
