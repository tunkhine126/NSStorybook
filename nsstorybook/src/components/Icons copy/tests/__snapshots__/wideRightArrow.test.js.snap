// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<WideRightArrow /> Should render and match the snapshot 1`] = `
<svg
  fill="none"
  height="60"
  viewBox="0 0 22 39"
  width="22"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M1 0L17 20.5128L1 40"
    stroke="#DADAE0"
  />
</svg>
`;
